mexico-city/Dockerfile

```diff
@@ -0,0 +1,6 @@
+# Note to self: Alpine Linux is the devil and should not be used
```

Comment:
Can you add the standard file header with copyright/licensing notice to the top of this file?

Reply:
done